modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/Sidebar.js

```diff
@@ -71,17 +71,19 @@ const Sidebar = ({children, onClose = noop, open = true}) => {
 		}
 	}, [delay, open]);
 
+	useEffect(() => {
+		const body = document.querySelector('body');
+
+		isOpen
+			? body.classList.add(`sidebar-open`)
+			: body.classList.remove(`sidebar-open`);
```

Comment:
I imagine the formatter will change the template literals to single quotes for you, but if it doesn't, you should change them.

Reply:
👍